app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people: {str, "Person"} = {}
+
+    def __init__(self, name: str = None, age: int = None) -> None:
+        self.name = name
+        self.age = age
+        Person.people.update({name: self})
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(person.get("name"),
+                          person.get("age")) for person in people]
+    for i in range(len(people)):
+        if people[i].get("wife"):
+            person_list[i].wife = Person.people.get(people[i].get("wife"))
+        if people[i].get("husband"):
+            person_list[i].husband = Person.people.get(
+                people[i].get("husband"))
```

Comment:
what does `i` variable mean? specify a better name, `person` for example. Also, you can iterate throught people:

```
for person in people
```